com.unity.perception/Documentation~/index.md

```diff
@@ -1,15 +1,19 @@
-# About the Perception SDK
-com.unity.perception provides a toolkit for generating large-scale datasets for perception-based machine learning training and validation. It is focused on a handful of camera-based use cases for now and will ultimately expand to other forms of sensors and machine learning tasks.
+<img src="images/banner2.PNG" align="middle"/>
 
-# Technical details
-## Requirements
+# com.unity.perception
```

Comment:
I don't know if we should use the package name like this, I was told to change this in a PR for perception to Perception SDK.

Reply:
Good call. @sannambhotla what do you think?

Reply:
For consistency, we should call it Perception package and we can cite it something like this:

Unity Perception package (com.unity.perception)?

Reply:
Updated. Thanks!